generators/server/files.js

```diff
@@ -1371,29 +1375,102 @@ const baseServerFiles = {
         },
       ],
     },
+    {
+      condition: generator => generator.databaseTypeSql,
```

Comment:
Would be nice to move every new sql file to `files-sql.js`

Reply:
Referenced for further actions: https://github.com/jhipster/generator-jhipster/issues/18758

Reply:
see https://github.com/Tcharl/generator-jhipster/tree/testcontainers_sql_rmshima